app/src/main/res/values/dimens.xml

```diff
@@ -18,217 +18,317 @@
   <dimen name="bottom_white_space">160dp</dimen>
   <dimen name="text_input_layout_corner_radius">4dp</dimen>
   <dimen name="text_input_layout_stroke_width">1dp</dimen>
+  <dimen name="default_reading_text_size">18sp</dimen>
 
-  <dimen name="clickable_item_min_height">48dp</dimen>
+  <dimen name="space_0dp">0dp</dimen>
+  <dimen name="dot_width_height">12dp</dimen>
+  <dimen name="dot_gap">12dp</dimen>
 
+  <!-- Ongoing Story Card View -->
   <dimen name="ongoing_story_card_view_is_tablet_margin_top">28dp</dimen>
   <dimen name="ongoing_story_card_view_margin_top_portrait_story_grid_greater_then_2">28dp</dimen>
   <dimen name="ongoing_story_card_view_margin_top_landscape">28dp</dimen>
+  <dimen name="ongoing_story_card_view_margin_top_portrait">16dp</dimen>
+
+  <!-- Font Scale -->
   <dimen name="font_scale_content_extra_large_text_view_size">28dp</dimen>
-  <dimen name="state_fragment_non_split_view_margin_end">28dp</dimen>
-  <dimen name="state_fragment_non_split_view_margin_start">28dp</dimen>
-  <dimen name="profile_chooser_add_view_margin_bottom_profile_not_added">28dp</dimen>
+  <dimen name="font_scale_content_small_text_view_size">16dp</dimen>
+  <dimen name="font_scale_content_large_text_view_size">24dp</dimen>
+  <dimen name="font_scale_content_text_size">20dp</dimen>
+
+  <!-- Profile Chooser Fragment -->
+  <dimen name="profile_chooser_fragment_profile_select_text_margin_top">64dp</dimen>
+  <dimen name="profile_chooser_fragment_profile_recycler_view_margin_top">36dp</dimen>
+  <dimen name="profile_chooser_fragment_margin_top_profile_already_added">24dp</dimen>
+
+  <!--  Profile Chooser Profile View -->
+  <dimen name="profile_chooser_profile_view_view_margin_top">60dp</dimen>
   <dimen name="profile_chooser_profile_view_margin_bottom_profile_not_added">28dp</dimen>
+  <dimen name="profile_chooser_profile_view_margin_start_profile_already_added">16dp</dimen>
+  <dimen name="profile_chooser_profile_view_margin_top_profile_already_added">16dp</dimen>
+  <dimen name="profile_chooser_profile_view_margin_bottom_profile_already_added">16dp</dimen>
+  <dimen name="profile_chooser_profile_view_margin">16dp</dimen>
+  <dimen name="profile_chooser_profile_view_margin_start">16dp</dimen>
+  <dimen name="profile_chooser_profile_view_margin_end">16dp</dimen>
+  <dimen name="profile_chooser_profile_view_margin_end_profile_already_added">24dp</dimen>
+  <dimen name="profile_chooser_profile_view_margin_top_profile_not_added">24dp</dimen>
+  <dimen name="profile_chooser_profile_view_margin_start_profile_not_added">24dp</dimen>
+  <dimen name="profile_chooser_profile_view_margin_end_profile_not_added">24dp</dimen>
+  <dimen name="profile_chooser_profile_view_divider_margin_top">40dp</dimen>
+  <dimen name="profile_chooser_profile_view_name_margin_top_profile_already_added">8dp</dimen>
+  <dimen name="profile_chooser_profile_view_circular_image_margin_top_profile_already_added">48dp</dimen>
 
```

Comment:
Nit: This newline is not needed since these dimens are in the same category

Reply:
okay removed